src/parser/jobs/blm/modules/Gauge.js

```diff
@@ -245,7 +256,7 @@ export default class Gauge extends Module {
 		this._enochianDownTimer.stop = 0
 	}
 
-	_renderLostFouls(time) {
+	_renderLostPolyglots(time) {
```

Comment:
render?

Reply:
Didn't really look at what the function did, since I was just changing the name now that there's more than one Polyglot spender, but you have a good point. Changed to be even more accurate (_countLostPolyglots)